lib/services/platform-service.ts

```diff
@@ -236,13 +236,21 @@ export class PlatformService extends EventEmitter implements IPlatformService {
 			platform = this.$mobileHelper.normalizePlatformName(platform);
 			this.$logger.trace("Validate options for platform: " + platform);
 			const platformData = this.$platformsData.getPlatformData(platform, projectData);
-			return await platformData.platformProjectService.validateOptions(projectData.projectId, provision, teamId);
+			return await platformData.platformProjectService.validateOptions(
+				projectData.projectIdentifiers[platform.toLowerCase()],
```

Comment:
why don't we just validate both project identifiers?

Reply:
Because its not relevant to validate android identifier when trying to run the application on iOS device,

Reply:
Codewise it kind of makes sense, but I get what you mean.